index.html

```diff
@@ -10,8 +10,10 @@
       <h1>Bus Mall</h1>
     </div>
     <div class="" id="content">
-
     </div>
+    <ul id="list">
```

Comment:
Technically, all uls are lists. Can you think of a more informative id for this element?